tests/backup_helper.go

```diff
@@ -8540,3 +8542,39 @@ func GetUpdatedKubeVirtVMSpecForBackup(scheduledAppContextsToBackup []*scheduler
 	}
 	return nil
 }
+
+// GetPVCList will get the list of PVC's and returns the PVC names
+func GetPVCList() ([]string, error) {
```

Comment:
Either make this function accept any namespace or name this function as `GetPVCListFromPxBackupNamespace()`
I would prefer the former.

Reply:
@mkoppal-px I have done the requested changes, please review.